api/src/opentrons/protocol_api/labware.py

```diff
@@ -215,8 +215,9 @@ def load_liquid(self, liquid: Liquid, volume: float) -> None:
         """
         Load a liquid into a well.
 
-        :param liquid: The type of liquid to load into the well.
-        :param volume: The volume of liquid to load, in µL.
+        :param liquid: a :py:class:`opentrons.types.Liquid`. The return type when defining a liquid in protocol_context.define_liquid.
+            The liquid to load into the well.
+        :param volume: a float that represents the volume of liquid to load, in µL.
```

Comment:
I think we should leave these as they were before. The function definition above indicates that it takes a Liquid and a float.

Reply:
Actually I just looked at the ticket and _this_ is the annotation we need:
```suggestion
        :param Liquid liquid: The liquid to load into the well.
        :param str volume: The volume of liquid to load, in µL.
```